src/sass/components/_footer.scss

```diff
@@ -84,3 +80,64 @@
     margin-top: 22px;
   }
 }
+
+@media (min-width: 768px) {
+  .footer {
+    ::after {
+      transform: translateY(-73px);
+    }
+    &__style {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+    }
+    &__header {
+      font-size: 26px;
+      line-height: 30px;
+      letter-spacing: 0.04em;
+      max-width: 340px;
+      margin-bottom: 91px;
```

Comment:
W tym wypadku odsuwasz się od krawędzi rodzica, więc to rodzic powinien mieć padding, który wpychałby swoje dzieci do środka.